css/src/components/message.scss

```diff
@@ -33,7 +35,7 @@ $message-border: $border-width solid $border !default;
 
 	.message-content {
 		grid-area: message-content;
-		padding: $message-spacer-lg;
+		padding: 1em;
```

Comment:
Are we sure we want `1em` here? This means that as text size increases the padding of the element also increases.

Reply:
I hard coded padding to 0.875rem for message-sm before and Olga recommended me to use 1em here to sync with font size (see last comment). I think that's a good solution, so I updated here.

Reply:
Please move the value into the variable:

```
$message-content-padding: 1em !default;
```

Reply:
Moved.

Reply:
I think we're better off here with rem for this particular rule. I don't think modification of size should be affecting message padding, see below for an extreme example (with font-size set to 50): 

<img width="379" alt="image" src="https://github.com/microsoft/atlas-design/assets/30843002/1eacdc60-65a7-4efa-a963-0a6f9e86feb1">


Compare to my suggestion (rem):

<img width="413" alt="image" src="https://github.com/microsoft/atlas-design/assets/30843002/0b127055-73de-4580-903a-a4424d10a3b8">